.drone.yml

```diff
@@ -91,7 +94,7 @@ steps:
       LOG_PASSWORD:
         from_secret: LOG_PASSWORD
     commands:
-      - export BRANCH=$(scripts/analysis/getBranchName.sh $GIT_USERNAME $GIT_TOKEN $DRONE_PULL_REQUEST)
```

Comment:
This script retrieves the base branch of this PR, not the current branch name.
Naming of the script is a bit misleading.

Reply:
> This script retrieves the base branch of this PR, not the current branch name. Naming of the script is a bit misleading.

For me it looks like it's not he base/parent branch.

The following command is used in the deleted script `getBranchName.sh`:

```
curl 2>/dev/null -u $1:$2 https://api.github.com/repos/nextcloud/talk-android/pulls/$3 | grep \"ref\": | grep -v master | cut -d"\"" -f4
```
The result for this PR:
```shell
$ curl 2>/dev/null https://api.github.com/repos/nextcloud/talk-android/pulls/2165 | grep \"ref\": | grep -v master | cut -d"\"" -f4
chore/noid/script-work
```

Also in the afterwards the usage of the base/parent branch makes no sense in my view. For example pushing to the parent/base branch in `findbugs-up.rb`:

https://github.com/nextcloud/talk-android/blob/chore/noid/script-work/scripts/analysis/findbugs-up.rb#L132

Reply:
I understood it wrong :)